src/pudl_archiver/metadata/sources.py

```diff
@@ -44,4 +145,121 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "usgsuswtdb": {
+        "title": "USGS USWTDB - U.S. Wind Turbine Database",
+        "path": "https://energy.usgs.gov/uswtdb/data/",
+        "description": (
+            "The United States Wind Turbine Database (USWTDB) provides the locations "
+            "of land-based and offshore wind turbines in the United States, "
+            "corresponding wind project information, and turbine technical "
+            "specifications. Wind turbine records are collected and compiled from "
+            "various public and private sources, digitized and position-verified from "
+            "aerial imagery, and quality checked. The USWTDB is available for download "
+            "in a variety of tabular and geospatial file formats, to meet a range of "
+            "user/software needs. Dynamic web services are available for users that wish "
+            "to access the USWTDB as a Representational State Transfer Services (RESTful) "
+            "web service."
+        ),
+        "working_partitions": {
+            "year_months": [
+                "2018-04",
+                "2018-07",
+                "2018-10",
+                "2019-01",
+                "2019-04",
+                "2019-07",
+                "2019-10",
+                "2020-01",
+                "2020-04",
+                "2020-05",
+                "2020-07",
+                "2020-10",
+                "2021-01",
+                "2021-04",
+                "2021-07",
+                "2021-11",
+                "2022-01",
+                "2022-04",
+                "2022-07",
+                "2022-10",
+                "2023-01",
+                "2023-05",
+                "2023-11",
+                "2024-05",
+                "2024-11",
+            ]  # these are almost year_quarters but not quite....
+        },
+        "keywords": sorted(
+            {
+                "usgs",
+                "uswtdb",
+                "wind",
+                "wind turbines",
+            }
```

Comment:
Add keywords indicating the geospatial nature of this dataset, and more general energy related keywords. Also for PV below.

```suggestion
            {
                "usgs",
                "uswtdb",
                "wind",
                "wind turbines",
                "spatial",
                "gis",
                "geospatial",
                "geometry",
                "renewable",
                "electricity",
                "generation",
                "energy",
            }
```